ansible/tasks/setup-supabase-internal.yml

```diff
@@ -47,18 +47,14 @@
     cmd: |
       usermod -a -G postgres vector
 
-- name: create vector overrides dir
-  file:
-    state: directory
-    owner: root
-    group: root
-    path: /etc/systemd/system/vector.service.d
-    mode: '0700'
-
-- name: Custom systemd overrides - ensure Vector gets restarted
-  copy:
-    src: files/vector.service.conf
```

Comment:
remove the corresponding file if it's not being used

Reply:
gotcha